repos/system_upgrade/el7toel8/actors/firewalldreadlockdownwhitelist/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.models import FirewalldLockdownWhitelist
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+from leapp.libraries.actor import private
+
+import xml.etree.ElementTree as ElementTree
+
+class FirewalldReadLockdownWhitelist(Actor):
+    """
+    Provides data about firewalld Lockdown Whitelist
+
+    After collecting data from the configuration file, a message with relevant
+    data will be produced.
+    """
+
+    name = 'firewalld_read_lockdown_whitelist'
+    consumes = ()
+    produces = (FirewalldLockdownWhitelist,)
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag)
```

Comment:
Usually such data collection should be done in the Facts collections phase. using the `FactsPhaseTag`.

Is there any particular reason why you do it so late?

Reply:
Nope. Just a mistake. I changed it to `FactsPhaseTag`.